pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/integrationLayer/ContentListView.kt

```diff
@@ -59,17 +64,51 @@ fun ContentListView(
         } else {
             LazyColumn(
                 modifier = modifier,
-                contentPadding = PaddingValues(horizontal = 16.dp),
-                verticalArrangement = Arrangement.spacedBy(8.dp)
+                contentPadding = PaddingValues(
+                    start = 16.dp,
+                    end = 16.dp,
+                    bottom = 16.dp
```

Comment:
Since you have aligned header with cell content, did you consider having constants which are reused consistently so that you could later update this value to, say `20.dp`, without having to do it in several locations?

_This would have been overkill_ is an acceptable answer 😅

Reply:
I thought about it, then continued with my work 😅 
I've created [this ticket](https://github.com/SRGSSR/pillarbox-android/issues/330) to address that, since it definitely makes sense 🙂